docs/11-circleci/02-activation.mdx

```diff
@@ -0,0 +1,135 @@
+and the latter to the _Pro_ and _Plus_ versions. Based on your use case proceed to the
+[Personal](#personal-license) or [Professional](#professional-license) license section.
+
+## Personal license
+
+Follow these steps if you are using Unity's _free_ version.
+
+> _**Note:** Do NOT follow the steps for the personal license if you have a professional license._
+
+#### Mental model
+
+1. Use the Unity orb to create an activation file.
+2. Convert the activation file to a license file.
+3. Encode the license file and save it in a CircleCI Context.
+
+#### Creating an activation file
+
+Add the workflow below to a [project set up in CircleCI](https://circleci.com/docs/getting-started).
+As long as you can access it in the CircleCI Dashboard and download its artifacts, it can be any
+project.
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  create-unity-activation-file:
+    jobs:
+      - unity/create-activation-file
+```
+
+If you require an activation file for a specific Editor version, provide its semantic version. The
+available options can be found on the [versions](/docs/docker/versions) page.
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  create-unity-activation-file:
+    jobs:
+      - unity/create-activation-file:
+          editor_version: 2022.1.11f1
+```
+
+Commit and push the new workflow and navigate to your project in the CircleCI Dashboard. Identify
+the pipeline triggered by your changes and open the `unity/create-activation-file` job. In the
+`ARTIFACTS` tab, you will encounter the `Unity.alf` file. Download it and follow the next section
+steps
+
+#### Converting the activation file
+
+1. Navigate to the [manual activation](https://license.unity3d.com/manual) page and upload the
+   `Unity.alf` file.
+2. Answer the prompted questions.
+3. Download the license file (Unity_v20XX.x.ulf). It's ok if the numbers don't match your exact
+   Unity version.
+
+#### Encoding the license
+
+Encode the contents of your `Unity_v20XX.x.ulf` file to base64. If you have the `base64` command
+line utility you can use:
+
+```
+cat Unity_v20XX.x.ulf | base64
+```
+
+#### Preparing the Context
+
+1. Navigate to [CircleCI](https://app.circleci.com/) ⇢ `Organization Settings` ⇢ `Contexts` ⇢
+   `Create Context` and choose a context name. Alternatively, open an existing Context that you want
+   to use.
+2. Create the following environment variables:
+   - `UNITY_ENCODED_LICENSE` - _(Paste the content from the
+     [Encoding the License](#encoding-the-license) section here)_
+   - `UNITY_USERNAME` - _(Add the email address that you use to log in to Unity)_
+   - `UNITY_PASSWORD` - _(Add the password that you use to log in to Unity)_
+3. Remove the job you added to your `.circleci/config.yml` on the
+   [Creating an Activation File](#creating-an-activation-file) section as it's no longer necessary.
```

Comment:
```suggestion
3. The job that you add before to get your activation file is no longer necessary. You can safely remove or disable it.
```

Reply:
All action items in this list start with a verb: navigate, create and remove. Changing this pattern for one item will break the reading flow. However, I can't think of a better alternative, so I'll make the change if you are set on it.

Reply:
Right. I am definitely not set on pushing for a change that doesn't make sense.

I was trying to get across that it is optional. I didn't think the link was necessary here reading it from the eyes of someone who has just implemented that.

Reply:
I agree the link is too much coming from the perspective of someone going through the docs. I'll reword what you suggested to fit the format of the list.